include/con4m/styledb.h

```diff
@@ -1,145 +1,145 @@
-bold_off(render_style_t *style)
+c4m_bold_off(c4m_render_style_t *style)
 {
-    style->base_style &= ~BOLD_ON;
+    style->base_style &= ~C4M_STY_BOLD;
 }
 
 static inline void
-italic_on(render_style_t *style)
+c4m_italic_on(c4m_render_style_t *style)
 {
-    style->base_style |= ITALIC_ON;
+    style->base_style |= C4M_STY_ITALIC;
 }
 
 static inline void
-italic_off(render_style_t *style)
+c4m_italic_off(c4m_render_style_t *style)
 {
-    style->base_style &= ~ITALIC_ON;
+    style->base_style &= ~C4M_STY_ITALIC;
 }
 
 static inline void
-strikethru_on(render_style_t *style)
+c4m_strikethru_on(c4m_render_style_t *style)
 {
-    style->base_style |= ST_ON;
+    style->base_style |= C4M_STY_ST;
 }
 
 static inline void
-strikethru_off(render_style_t *style)
+c4m_strikethru_off(c4m_render_style_t *style)
 {
-    style->base_style &= ~ST_ON;
+    style->base_style &= ~C4M_STY_ST;
 }
 
 static inline void
-underline_off(render_style_t *style)
+c4m_underline_off(c4m_render_style_t *style)
 {
-    style->base_style &= ~(UL_ON | UL_DOUBLE);
+    style->base_style &= ~(C4M_STY_UL | C4M_STY_UUL);
 }
 
 static inline void
-underline_on(render_style_t *style)
+c4m_underline_on(c4m_render_style_t *style)
 {
-    underline_off(style);
-    style->base_style |= UL_ON;
+    c4m_underline_off(style);
+    style->base_style |= C4M_STY_UL;
 }
 
 static inline void
-double_underline_on(render_style_t *style)
+c4m_double_underline_on(c4m_render_style_t *style)
 {
-    underline_off(style);
-    style->base_style |= UL_DOUBLE;
+    c4m_underline_off(style);
+    style->base_style |= C4M_STY_UUL;
 }
 
 static inline void
-inverse_on(render_style_t *style)
+c4m_reverse_on(c4m_render_style_t *style)
 {
-    style->base_style |= INV_ON;
+    style->base_style |= C4M_STY_REV;
 }
 
 static inline void
-inverse_off(render_style_t *style)
+c4m_reverse_off(c4m_render_style_t *style)
 {
-    style->base_style &= ~INV_ON;
+    style->base_style &= ~C4M_STY_REV;
 }
 
 static inline void
-casing_off(render_style_t *style)
+c4m_casing_off(c4m_render_style_t *style)
 {
-    style->base_style &= ~TITLE_CASE;
+    style->base_style &= ~C4M_STY_TITLE;
 }
 
 static inline void
-lowercase_on(render_style_t *style)
+c4m_lowercase_on(c4m_render_style_t *style)
 {
-    casing_off(style);
-    style->base_style |= LOWER_CASE;
+    c4m_casing_off(style);
+    style->base_style |= C4M_STY_LOWER;
 }
 
 static inline void
-uppercase_on(render_style_t *style)
+c4m_uppercase_on(c4m_render_style_t *style)
 {
-    casing_off(style);
-    style->base_style |= UPPER_CASE;
+    c4m_casing_off(style);
+    style->base_style |= C4M_STY_UPPER;
 }
 
 static inline void
-titlecase_on(render_style_t *style)
+c4m_titlecase_on(c4m_render_style_t *style)
 {
-    casing_off(style);
-    style->base_style |= TITLE_CASE;
+    c4m_casing_off(style);
+    style->base_style |= C4M_STY_TITLE;
 }
 
-extern const border_theme_t *registered_borders;
+const extern c4m_border_theme_t *c4m_registered_borders;
```

Comment:
The `extern` / `const` order swap was lost from `main`. The `const` before `extern` triggers an error from gcc on Linux.